src/steth/PriceConverter.sol

```diff
@@ -80,6 +85,30 @@ contract PriceConverter is AccessControl {
         return (_usdcAmount * C.WETH_USDC_DECIMALS_DIFF).mulWadDown(uint256(usdcPriceInEth));
     }
 
+    function ethToDai(uint256 _ethAmount) internal view returns (uint256) {
+        (, int256 daiPriceInEth,,,) = daiToEthPriceFeed.latestRoundData();
+
+        return _ethAmount.divWadDown(uint256(daiPriceInEth));
+    }
+
+    function daiToEth(uint256 _daiAmount) internal view returns (uint256) {
+        (, int256 daiPriceInEth,,,) = daiToEthPriceFeed.latestRoundData();
+
+        return _daiAmount.mulWadDown(uint256(daiPriceInEth));
+    }
```

Comment:
there is no need for these internal functions, you can inline them where required

Reply:
There is no hurt right. I think it makes the code easier to read.

Reply:
i'm not quite sure about that in this case, but it's your call. at least add "_" prefix to these functions so they respect the convention used elsewhere in the code